google/provider/provider_mmv1_resources.go

```diff
@@ -519,8 +519,8 @@ var handwrittenIAMDatasources = map[string]*schema.Resource{
 
 // Resources
 // Generated resources: 596
-// Generated IAM resources: 297
```

Comment:
@c2thorn I think the IAM resources are only added to beta. Can you verify if the number change is intended?